django/contrib/admin/static/admin/img/README.txt

```diff
@@ -1,7 +1,15 @@
-All icons are taken from Font Awesome (https://fontawesome.com/) project.
-The Font Awesome font is licensed under the SIL OFL 1.1:
-- https://scripts.sil.org/OFL
+## Icons License
 
-SVG icons source: https://github.com/encharm/Font-Awesome-SVG-PNG
-Font-Awesome-SVG-PNG is licensed under the MIT license (see file license
-in current folder).
+All icons in this directory are provided by [Font Awesome Free](https://fontawesome.com), version 6.7.1
```

Comment:
I don’t think that’s the case but could be wrong? The files in the `gis` subdirectory don’t look like Font Awesome to me. I could also see in the future us adopting other icons and placing them in this folder as it’s a generic `img` folder, not `fontawesome`.